types/sumologic/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for sumologger 1.0
+// Project: https://github.com/helix-collective/node-sumologic
+// Definitions by: banadiga <https://github.com/banadiga>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare class SumoLogger {
+    constructor(collectorCode: string, opts?: object);
+
+    replaceConsole(): void;
+    restoreConsole(): void;
+    augmentConsole(): void;
+    log(): void;
```

Comment:
This seems incorrect definition.. as seems like it accepts string as per https://github.com/helix-collective/node-sumologic#usage